app/Helpers/DateHelpers/Helper.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Helpers\DateHelpers;
```

Comment:
Unnecessary nesting. Let it be just a class `DateHelper` within `App\Helpers` namespace.